spark/v3.2/spark/src/main/java/org/apache/iceberg/spark/procedures/AddFilesProcedure.java

```diff
@@ -127,6 +135,11 @@ private boolean isFileIdentifier(Identifier ident) {
 
   private long importToIceberg(Identifier destIdent, Identifier sourceIdent, Map<String, String> partitionFilter,
```

Comment:
Is this still used now?

Reply:
It seems that not been used anymore,  I will remove it